.flake8

```diff
@@ -8,4 +8,4 @@ exclude =
     coverage,
     node,
     node_modules
-max-line-length=88
+max-line-length=100
```

Comment:
Is this a change we want to make? We should make a matching change where we call `black` then. I'll push that.